recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
   run:
     - python
     - pytest >=7.0.0
+    - py
```

Comment:
```suggestion
```
`py` was removed in v2.0.3., see https://github.com/pytest-dev/pytest-metadata/blob/3.0.0/CHANGES.rst#203-2022-10-26